packages/ndla-ui/src/Search/ContentTypeResult.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState, useEffect, useRef } from 'react';
```

Comment:
Denne filen blir nok litt for lang. Her har du muligheter til å splitte opp listen og legge liste items i en egen komponent.

Reply:
Ja, deler den opp:)